docs/zh_CN/toolkit/nftp.md

```diff
@@ -0,0 +1,75 @@
+# NFTP
+
+nftp 是基于MQTT的轻量级文件传输工具。
+
+## 编译 
+
+**注意**： nftp 工具默认不构建，您可以通过`-DBUILD_NFTP=ON` 启用它。
+
+```bash
+$ cmake -G Ninja -DBUILD_NFTP=ON ..
+$ Ninja
+```
```

Comment:
Hmmm should we go to build folder first？

Reply:
??